docs/podman-rmi.1.md

```diff
@@ -1,9 +1,11 @@
 % podman-rmi(1)
 
 ## NAME
-podman\-rmi - Removes one or more images
+podman\-image\-rm (podman\-rmi) - Removes one or more images
 
 ## SYNOPSIS
+**podman image rm** *image* ...
```

Comment:
rmi not rm no?

Reply:
The command is `podman image rm`, This is correct.

Reply:
When I do:  `podman image --help`  it lists 
`  rmi         Removes one or more images from local storage`
So maybe the inner workings now respond to `image rm`, but at least Cobra help thinks `image rmi`.